services/trustdidweb-server-py/charts/dev/values.yaml

```diff
@@ -1,23 +1,21 @@
 ---
-ingressSuffix: .apps.silver.devops.gov.bc.ca
-
 ingress:
-  labels:
-    certbot-managed: "true"
   annotations:
     route.openshift.io/termination: edge
 networkPolicy:
   ingress:
     namespaceSelector:
       network.openshift.io/policy-group: ingress
-      
+
 server:
   image:
+    repository: image-registry.openshift-image-registry.svc:5000/4a9599-tools/tdw-server
+    tag: latest
```

Comment:
I forgot to pluck this out - we should be able to remove and just use what is defined in the default `values.yaml` (this was just so we could test changes without the need for a new release). Do you mind removing the repository/tag reference @PatStLouis ?